File: LeetCode/longestCommonPrefix.js
const longestCommonPrefix = strs => {
    let res = '';
    const firstStr = strs[0];
    if (strs.length === 1) {
        return firstStr;
    }
    if (firstStr === '') {
        return '';
    }
    for (let j = 0; j < firstStr.length; j++) {
        const l = firstStr[j];
        for (let i = 1; i < strs.length; i++) {
            const str = strs[i];
            if (str === '') {
                return '';
            }
            if (str[j] !== l) {
                return res;
            }
        }
        res += l;
    }  
    return res;
};

File: LeetCode/DailyDoseCSharp/LongestCommonPrefix.cs
public class Solution {
    public int bb;
    public string LongestCommonPrefix(string[] strs) {
        var b = nameof(Aname);
        var commenPrefix = "";
        for (int i = 0; i < strs[0].Length; i++)
        {
            char c = strs[0][i];
            for (int j = 1; j < strs.Length; j++)
            {
                if (strs[j].Length <= i || strs[j][i] != c)
                {
                    return commenPrefix;
                }
            }
            commenPrefix += c;
        }
        return commenPrefix;
    }
}